.github/workflows/rust.yml

```diff
@@ -129,6 +129,7 @@ jobs:
 
   calibnet-check:
     name: Calibnet sync check
+    if: ${{ false }}
```

Comment:
Do we want to disable snapshot fetch and import test as well?

Reply:
Perhaps it's worth leaving those steps enabled, good point. Not ideal but at least something would be covered.

Reply:
Where are those tests?

Reply:
https://github.com/ChainSafe/forest/pull/2093/files#diff-73e17259d77e5fbef83b2bdbbe4dc40a912f807472287f7f45b77e0cbf78792dR155-R163

Reply:
I'm not sure the link works. In general, commenting out everything below `forest --chain calibnet --target-peer-count 50 --encrypt-keystore false --halt-after-import --import-snapshot /tmp/snapshots/*.car` because up until this point it should still pass, right? And at least we would be able to detect import/fetch fiascoes.

Reply:
Not sure that links for me for. What is it supposed to show?